client/src/document/index.scss

```diff
@@ -326,9 +326,13 @@ iframe {
   &.nobutton,
   &.sample-code-frame {
     background: #fff;
-    border: 1px solid var(--border-primary);
-    border-radius: var(--elem-radius);
-    width: 100%;
+    border: none;
+    border-radius: calc(var(--elem-radius) / 2);
+    box-shadow: 0 0 1px 0.25rem #fff;
+    margin-left: 0.25rem;
+    outline: 1px solid var(--border-primary);
+    outline-offset: 0.25rem;
+    width: calc(100% - 0.5rem);
```

Comment:
I wasn't opposed to these changes, even though it looked a bit "hacky". Now I remembered and checked that adding `box-sizing: content-box` would have the same effect. Have you considered this and/or are there any downsides of using [box-sizing](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing)?

```suggestion
    border: 1px solid var(--border-primary);
    border-radius: var(--elem-radius);
    box-sizing: content-box;
    width: 100%;
```

Reply:
Yes, I had considered that. But, as mentioned in the OP, I wanted to create similar padding effect that we had before.
Because without it contributors may add `margin`, `margin-top`, `padding`, or `padding-top` themselves. Something like:
````
```css hidden
body {
  padding: 1rem;
  /* padding-top: 1rem; */
}

#myElement {
  margin: 1rem;
  /* margin-top: 1rem; */
}
```
````
Not necessarily in hidden blocks but anywhere to make the output look decent.

Some PRs for reference: https://github.com/mdn/content/pull/19176/files, https://github.com/mdn/content/pull/17904/files,  https://github.com/mdn/content/pull/18054/files

***
I wish we had taken the other route of fixing it in the content (https://github.com/mdn/content/pull/16998).
Now most of the outputs [look glued to the borders](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#simple_label_example):   :roll_eyes: 
![_glued](https://user-images.githubusercontent.com/87750369/188271920-8029ad6c-c38c-4032-8f95-4235c7fb8721.png)